geodeForRedis/src/main/java/org/apache/geode_examples/geodeForRedis/Example.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode_examples.geodeForRedis;
+
+import java.text.DecimalFormat;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+
+import redis.clients.jedis.HostAndPort;
+import redis.clients.jedis.JedisCluster;
+
+public class Example {
+  public static final String SORTED_SET_KEY = "leaderboard";
+
+  public static void main(String[] args) {
+    JedisCluster jedis = new JedisCluster(new HostAndPort("127.0.0.1", 6379));
+
+    populateSortedSet(jedis);
+
+    System.out.println("Initial leaderboard with key '" + SORTED_SET_KEY + "': "
```

Comment:
Since the "jedis.zrangeWithScores(SORTED_SET_KEY, 0, Integer.MAX_VALUE)" is repeated, possibly make it a method, say:

private static void printSortedSet(JedisCluster jedis, String message) {
         System.out.println(message + ": " +  jedis.zrangeWithScores(SORTED_SET_KEY, 0, Integer.MAX_VALUE));
}

Very minor benefits, though.